src/modules/project/useCases/updateNewRulesOptIn.spec.ts

```diff
@@ -72,4 +79,26 @@ describe('ProjectSteps.updateNewRulesOptIn()', () => {
       expect(targetEvent.payload.projectId).toEqual(projectId)
     })
   })
+
+  describe(`Lorsque le projet est déjà souscrit au nouveau CDC`, () => {
+    it('doit retourner une erreur NouveauCahierDesChargesDéjàSouscrit', async () => {
+      fakePublish.mockClear()
+
+      const shouldUserAccessProject = jest.fn(async () => false)
```

Comment:
Pourquoi dire que le `shouldUserAccesProject` retourne false ? Dans le cadre de ce test on a envie qu'il soit true non?

```suggestion
      const shouldUserAccessProject = jest.fn(async () => true)
```

Reply:
Oui bien vu ! En plus ça m'a fait remarquer que le check dans le usecase devrait être dans l'autre sens